packages/sdk-react/src/components/receivables/InvoiceDetails/CreateReceivable/sections/components/ReminderSection/SelectFieldWithEdit.tsx

```diff
@@ -0,0 +1,78 @@
+import { ControllerRenderProps, FieldValues } from 'react-hook-form';
+
+import { RHFSelectField } from '@/components/RHF/RHFSelectField';
+import { t } from '@lingui/macro';
+import { useLingui } from '@lingui/react';
+import { Button, Grid, MenuItem, SelectChangeEvent } from '@mui/material';
+
+import { ReminderDetail, ReminderDetails } from './ReminderDetail';
+
+interface CustomSelectFieldProps extends FieldValues {
+  handleSelectChange: (
+    field: ControllerRenderProps<FieldValues, string>
+  ) => (event: SelectChangeEvent<string | number>) => void;
+  details: ReminderDetail | undefined;
+  options: Array<{ id: string | number; name: string }>;
+}
+
+export const SelectFieldWithEdit = ({
+  field,
+  label,
+  noOptionsText,
+  disabled,
+  options,
+  handleSelectChange,
+  control,
+  details,
+  createOptionLabel,
+}: CustomSelectFieldProps) => {
+  const { i18n } = useLingui();
+
+  return (
+    <Grid container alignItems="center" spacing={1}>
+      <Grid item xs={10}>
+        <RHFSelectField
```

Comment:
As far as I remember, we agreed you could use `RHFAutocomplete` here.
If you want to implement adding a new item, please check [the official documentation](https://mui.com/material-ui/react-autocomplete/#creatable)

Reply:
Done, refactored to RHFAutocomplete approach